integration_tests/mockApis/visitScheduler.ts

```diff
@@ -203,6 +203,50 @@ export default {
       },
     })
   },
+  stubBookedVisit: (reference: string): SuperAgentRequest => {
+    const result: Visit = {
```

Comment:
I don't think you actually need this new stub because one already exists for this endpoint (`stubVisit`) - it was added ages ago but isn't used I don't think. So, it could be tidied up and you could just pass in the `Visit` object for it to return rather than just the reference so it's more flexible.

(also, this endpoint on the API can return cancelled visits so `stubVisit` a more accurate name)